Portfolio.html

```diff
@@ -14,24 +14,24 @@
 <body>
 	<nav>
 		<ul>
-			<li><a href="home">Home</a></li>
-			<li><a href="portfolio">Portfolio</a></li>
-			<li><a href="aboutme">About Me</a></li>
-			<li><a href="contact">Contact</a></li>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
 		 </ul>
 	</nav>
 
 
 
-	<div class="intropage">
+	<div class="intropage" id="home">
 		<div class="container">
 			<h1>JENNY ALLTON-TWI<b>S</b>T</h1>
 		</div>
 	</div>
 
 
 
-	<div class="portfolio"><h2>Portfolio</h2>
+	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
 		<div class="container">
 			<div class="box">
```

Comment:
I'd probably suggest changing the background colour of your box class to offset it from the background slightly. I'd probably look at having less generic class names early on so you don't end up in trouble later.